docs/Coiote_IoT_DM/Coiote_videos/How_to_remotely_configure_IoT_devices.md

```diff
@@ -0,0 +1,17 @@
+# How to remotely configure IoT devices?
+
+With IoT devices enabled with LwM2M protocol, you can use a set of standardized remote operations to configure your IoT devices remotely.
+
+To prepare your device with LwM2M connectivity, grab Anjay IoT SDK, Zephyr Anjay Client, or any LwM2M Client. Check out more handy links below.
+
+In our tutorial, we'll be exploring remote configuration options such as Read, Write, Observe, and Execute operations.
+
+Connect your IoT devices with Anjay IoT SDK - LwM2M Client:
+
+* [Anjay Client repository](https://github.com/AVSystem/Anjay)
+* [Anjay Zephyr Client repository](https://github.com/AVSystem/Anjay-zephyr-client)
```

Comment:
Why only those two links?

Reply:
well on YouTube we also added the Devzone link but for Devzone I think we can have these two links. Overall, we wanted to highlight this configuration which we have in the video.